vm-protection/version-1/permission-group-SNAPSHOT_PRIVATE_ACCESS.json

```diff
@@ -7,12 +7,12 @@
         "scope": "subscription"
```

Comment:
why are we using the old structure for `vm-protection/version-1/permission-group-SNAPSHOT_PRIVATE_ACCESS.json`?

Reply:
This file was recently added in ce4a646e4e5f6f44ff026b7882fd39345da0b95f. Upon re-enabling the parity tests, I noticed that this file contains invalid permissions which differ from the SNAPSHOT_PRIVATE_ACCESS V1 permissions as stored in RSC and have included the fix in this PR.